registry/handlers/app.go

```diff
@@ -184,6 +184,18 @@ func NewApp(ctx context.Context, config *configuration.Configuration) *App {
 		}
 	}
 
+	// configure tag lookup concurrency limit
+	if p := config.Storage.TagParameters(); p != nil {
+		l, ok := p["concurrency_limit"]
```

Comment:
Every new contributor tries to add `snake_case` config fields for some reason. You can't do that.
https://github.com/distribution/distribution/blob/df983747649396a7377a4b428ad3a1aebf7a9001/configuration/configuration.go#L16-L17
```suggestion
		l, ok := p["concurrencylimit"]
```